bin/citgm-all.js

```diff
@@ -54,6 +54,11 @@ if (!lookup) {
   process.exit(1);
 }
 
+if (app.autoParallel) {
+  app.parallel = require('os').cpus().length;
```

Comment:
I think that it already is? https://github.com/GeorgeAdams95/citgm/blob/8526bb86dffb7c90bfd8d633b3c5d5e6e1fd884b/bin/citgm-all.js#L4